src/safeposix/syscalls/net_calls.rs

```diff
@@ -3103,31 +3103,69 @@ impl Cage {
         }
     }
 
+    /// ## ------------------GETPEERNAME SYSCALL------------------
+    /// ### Description
+    /// The `getpeername_syscall()` returns the address of the peer connected to
+    /// the socket fd, in the buffer pointed to by ret_addr
+
```

Comment:
Does Rustdoc behave in a sane way with blank lines interspersed?  I don't know...

Maybe make the blank lines have /// as well.